### DISCUSSION TITLE
Setup of zkSync node locally on WSL

### DISCUSSION THREAD

**Discussion by mishraji874** [Cyfrin/foundry-full-course-cu#1921] [Q&A] [answered]
Heyy everyone!! How can I start the zkSync node locally on wsl using docker?
Please try to help me with this. I wanted to work in zkSync but I can't able to start the node locally on the desktop.
Thank you in advance.

**Comment by EngrPips** [comment, marked as answer]
Honestly, I will say maybe look up stuff on the internet on how to start zkSync node locally on wsl using docker. I haven't done anything with Docker yet. Maybe someone that knows will help soon.

**Reply by AllenGeorge08** [reply]
Hey @mishraji874 ..You can install docker from here for wsl ( change acc to your distribution)....

(https://docs.docker.com/engine/install/debian/#install-using-the-repository)

And here is how you run the nodes

(https://docs.zksync.io/zksync-node/quickstart)

**Reply by EngrPips** [reply]
Awesome, Thanks for the heads-up @AllenOps08
